socorro/processor/processor_pipeline.py

```diff
@@ -150,33 +95,61 @@ class ProcessorPipeline(RequiredConfig):
             '{dump_file_pathname}'
         )
     )
-    required_config.add_option(
+    required_config.breakpad.add_option(
         'kill_timeout',
         doc='amount of time to let mdsw run before declaring it hung',
         default=600
     )
-    required_config.add_option(
+    required_config.breakpad.add_option(
         'symbol_tmp_path',
         doc=(
             'directory to use as temp space for downloading symbols--must be '
             'on the same filesystem as symbols-cache'
         ),
         default=os.path.join(tempfile.gettempdir(), 'symbols-tmp'),
     ),
-    required_config.add_option(
+    required_config.breakpad.add_option(
         'symbol_cache_path',
         doc=(
             'the path where the symbol cache is found, this location must be '
             'readable and writeable (quote path with embedded spaces)'
         ),
         default=os.path.join(tempfile.gettempdir(), 'symbols'),
     )
-    required_config.add_option(
-        'temporary_file_system_storage_path',
+    required_config.breakpad.add_option(
+        'tmp_storage_path',
         doc='a path where temporary files may be written',
         default=tempfile.gettempdir(),
     )
-    required_config.add_option(
+
+    # JitClassCategorizationRule configuration
+    required_config.jit = Namespace()
+    required_config.jit.add_option(
+        'kill_timeout',
+        doc='amount of time to let command run before declaring it hung',
+        default=600
+    )
+    required_config.jit.add_option(
+        'dump_field',
+        doc='the default name of a dump',
+        default='upload_file_minidump',
+    )
+    required_config.jit.add_option(
+        'command_pathname',
+        doc='full pathname to external program; quote path with embedded spaces',
+        default='/stackwalk/jit-crash-categorize'
+    )
+    required_config.jit.add_option(
+        'command_line',
+        doc='template for command line; uses Python format syntax',
+        default=(
+            'timeout -s KILL {kill_timeout} {command_pathname} {dump_file_pathname}'
+        )
+    )
```

Comment:
Previously, the `JitCrashCategorizeRule` was picking up the wrong configuration, but because it ran before signature generation, it never did anything so we never noticed. Yay.

This puts the configuration in its own namespace so it'll work now.